templates/prompt-for-mfa-manager.php

```diff
@@ -20,6 +20,9 @@
     <p>
         Enter code: <input type="text" id="mfaSubmission" name="mfaSubmission" />
         <br />
+        <input type="checkbox" name="rememberMe" id="rememberMe" value="true" checked="checked"/>
```

Comment:
Had this simply been left out when the manager option was originally added?

Reply:
Yes, so it's clumsy of me to sneak it in on an unrelated PR.

Reply:
Okay, not a problem. Just wanted to check. I've done the same (including a small unrelated change in a PR).